src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectDetailsAjaxController.java

```diff
@@ -162,4 +163,23 @@ public ProjectDetailsAjaxController(ProjectService projectService, UIProjectsSer
 					.body(new AjaxErrorResponse(e.getMessage()));
 		}
 	}
+
+	/**
+	 * Delete a project
+	 *
+	 * @param projectId identifier for a project
+	 * @return an indication to the user about the result of the update
+	 */
+	@DeleteMapping("/delete")
```

Comment:
Since it's a DeleteMapping we shouldn't need the "/delete"

Reply:
Sounds good, removed in a678205